swtr/server.py

```diff
@@ -63,15 +79,15 @@ def index():
     #print auth_tok
     payload = {'what': 'img-anno',
                'access_token': auth_tok['access_token']}
-    req = requests.get(config.swtstoreURL + '/api/sweets/q', params=payload)
+    req = requests.get(current_app.config.get('SWTSTORE_URL', 'SWTSTORE_URL') + '/api/sweets/q', params=payload)
     sweets = req.json()
     return render_template('index.html', access_token=auth_tok['access_token'],
                            refresh_token=auth_tok['refresh_token'],
-                           config=config, url=request.args.get('where'),
-                           bookmark=quote_plus(config.app_url), sweets=sweets)
+                           config=current_app.config, url=request.args.get('where'),
+                           bookmark=quote_plus(current_app.config.APP_URL), sweets=sweets)
```

Comment:
Sure! One test instance of swtstore is hosted at: http://teststore.swtr.us
You can use that, in your config file, in the SWTSTORE_URL use that URL, and it should work.

PS: Getting all SWTs from the store is open, only posting to them requires the client application to register.